packages/safety-suite-api/src/api/dashboardSink/kpi.ts

```diff
@@ -0,0 +1,39 @@
+import {Request} from '@idelic/safety-net';
+
+import {runApi} from '../../runApi';
+import {ApiOptions, ApiSuccessResponse} from '../../types';
+import {KpiQuery} from './types';
+
+export interface KpiGroup {
+  name: string;
+  fields: string[];
+  values: unknown[];
+  humanName: string;
+}
+
+export interface KpiMetadata {
+  query: any;
+  page: number;
+  groups: KpiGroup[];
+}
+
+export interface KpiResponse<Data = unknown> {
+  meta: KpiMetadata;
+  data: Record<string, Data[][]>;
```

Comment:
I think this should be `Record<string, [string, Data][]>;`.  I think that Data will always be `number`.
If that's not valid syntax, then something like:
```
type KpiData = [string, number]
export interface KpiResponse ... {
  ...
  data: Record<string, KpiData[]>;
}
```

Or similar.  I'm sure @maxswa knows this better than me.

Reply:
Thats a good point. I think the the first index will be string or null and the second will be number. So maybe something like 
```
type KpiData = [string | null, number]
```
I'll take a look.

Reply:
Ah that's right.